SQL/proc_descriptions/pgtap.md

```diff
@@ -0,0 +1,207 @@
+# Notes regarding pgTap testing framework
+
+## Instalation
+1) pgTAP must be installed on a host with PostgreSQL server running; it cannot be installed remotely. If you’re using PostgreSQL in Docker, you need to install pgTAP inside the Docker container.
+    If you are using Linux, you may (depending on your distribution) be able to use you distribution’s package management system to install pgTAP. For instance, on Debian, Ubuntu, or Linux Mint pgTAP can be installed with the command:
+    `sudo apt-get install pgtap`
+    On other systems pgTAP has to be downloaded and built. First, download pgTAP from PGXN (click the green download button in the upper-right). Extract the downloaded zip file, and (at the command line) navigate to the extracted folder.
+
+    To build pgTAP and install it into a PostgreSQL database, run the following commands:
+
+    ``` sh
+    make
+    make install
+    make installcheck
+    ```
+
+## Useful Notes
```

Comment:
I think some quick start guide should follow here, not notes

Reply:
Updated in commit __c59faf2__

Reply:
I think it should be still better written & structured so that developers can easily start to develop new tests without investing much time into understanding pgTap

Reply:
Any ideas for improvement?

Reply:
Rewritten existing part in commit `6bd2f03`, but no idea what else should I add to this quickstart guide

Reply:
I think it will be best to test it on some of your colleagues then. We can leave it like this, merge it, and improve it later on -demand...

Reply:
Yeah, I think it is a good idea, although we should warn them about incompleteness of the guide and ask them about leaving some comments of what was difficult to understand outright

Reply:
I agree